values.yaml

```diff
@@ -153,6 +153,12 @@ fernetKeySecretName: ~
 # Enable security context constraints required for OpenShift
 sccEnabled: false
 
+# Enable nginx auth sidecar
+authSidecar:
+  enabled: false
+  image: nginxinc/nginx-unprivileged
```

Comment:
I feel like this should be moved to `images.authSidecar.repository`, and also expose support for `tag` and `pullPolicy` too.

Reply:
repository and tags are added below the authSidecar section.
@danielhoherd - discussed with daniel to maintain all the values as a part of platform configuration

Reply:
The biggest consideration I have for where this should live is who has access to change the image. We want to make sure the image is not configurable by anybody who does not have privileges to launch arbitrary pods with escalated access.

Reply:
That makes sense. Is there some magic that allows user changes to `images` but not other helm parameters?